test/async-hooks/test-graph.tcp.js

```diff
@@ -17,7 +17,7 @@ const server = net
 
 server.listen(common.PORT);
 
-net.connect({ port: server.address().port, host: server.address().address },
+net.connect({ port: server.address().port, host: '::1' },
```

Comment:
Does that apply? Doesn't windows have an equivalent of `/etc/hosts` that it just looks up to find the address? I'm sure we're using `localhost` in a bunch of other tests.